### ISSUE TITLE
Keyboard focus loss is observed when we press TAB key after "Tree view": A11y_.NET CoreWinforms_Common_Controls2_WebBrowser_Keyboard

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#11437]
### .NET version

dotnet-sdk-9.0.100-preview.5.24258.1

### Did it work in .NET Framework?

Yes

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev3; #BM_.NET Core[Winforms]_Win32_May2024; #.NETCore; #WCAG2.4.7; #DesktopApp; #Win11; #FTP; #Keyboard;

### Environment Details:
App name: .NET Core Winforms
Window Version: Win 11
OS Build: 22621.3085

### Repro Steps:

1. Launch the "Accessibility core app".
2. Main Form screen would appear.
3. TAB to "Common_Controls2:Testing the controls under the Common_controls" and hit ENTER key.
4. Navigate using TAB key after the tree view.
5. Observe the issue.

### Actual Result:
Keyboard focus loss is observed when we press TAB key after "Tree view".

### Expected Result:
Keyboard focus should be clearly visible on scrollbar present for "Web browser" control when we navigate using TAB key after the "Tree view".

### User Impact:
Keyboard users will not be able to identify where the focus is and will not be able to interact with the control.

### Attachment

https://github.com/dotnet/winforms/assets/90901519/071400ba-1e04-4c9a-93ef-94e7207b1e96

**Comment by Olina-Zhang** [comment]
This is the WebBrowser with a simple html doc we updated last time to avoid web page in WebBrowser steals focus, see discussion in https://github.com/dotnet/winforms/issues/7337#issuecomment-1165930875

**Comment by merriemcgaw** [comment]
This is a Sev3 issue. We can close it on our end.

**Comment by Yash14j** [comment]
Github Tags:#Rev:yaja;
